tests/acceptance/tests/Settings/RuleBuilderCreate.spec.ts

```diff
@@ -0,0 +1,87 @@
+import { test } from '@fixtures/AcceptanceTest';
+
+test('As an admin user, I want to create a rule', { tag: '@Rule' }, async ({
+    AdminRuleDetail,
+    AdminRuleListing,
+    ShopAdmin,
+    IdProvider,
+    TestDataService,
+    CreateRule,
+}) => {
+
+    const { uuid: ruleId, id: uniqueId } = IdProvider.getIdPair();
+    const {id: taxId, name: taxName} = (await TestDataService.createTaxRate());
+    const today = new Date();
+    const yesterday = new Date();
+    yesterday.setHours(today.getHours() - 30);
+
+    const testConfig = {
+        ruleId,
+        ruleName: `Test rule - ${uniqueId}`,
+        ruleTypes: ['Price', 'Shipping', 'Payment', 'Flow Builder'],
+        rulePriority: 1,
+        ruleDescription: 'This is a test rule, created to test the Rule Builder.',
+        ruleTag: (await TestDataService.createTag(`Test tag - ${uniqueId}`)).name,
+        taxId,
+        taxName,
+        customerSurname: 'Schmitz-Rimpler',
+        fromDate: yesterday.toISOString().split('.')[0] + '+00:00',
+        toDate: today.toISOString().split('.')[0] + '+00:00',
+        quantity: 5,
+        isAdminOrder: false,
+        stock: 10,
+    };
+
+    await test.step('Create rule via API', async () => {
+        await ShopAdmin.attemptsTo(CreateRule(testConfig));
```

Comment:
I am wondering, if we do something similar also somewhere else. Because the way how we structure our wording here, one would probably think, that "ShopAdmin attempts to create Rule" would mean, that the "user" will make use of the UI to create rule, which is not the case here, as the task is sending an API request.
I would have rather expected it to be part of the TestDataService then, if we create entities via an API call.

@frobel what do you think of it?